src/main/java/org/kaazing/netx/ws/internal/io/OutgoingSentinelExtension.java

```diff
@@ -288,4 +291,18 @@ else if (reasonLength > 123) {
         }
     }
 
+    private void createMask(byte[] maskBuf) {
+        int m = maskInt.incrementAndGet();
```

Comment:
Is the mask predictable?  If so, then a malicious application can control the bytes on the wire, whereas the mask is intended to prevent such predictability.